docs/xdebug.md

```diff
@@ -8,16 +8,21 @@ This document describes how to enable Xdebug for local PHP development using the
    `docker-compose down` \
    `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d`
 
-   This version of `docker-compose up` uses an override file to modify our services. `docker-compose.dev.yml` we now have an extra PHP container called `esmero-php-debug`. To stop the containers in the future, run `docker-compose -f docker-compose.yml -f docker-compose.dev.yml down`.
+   <br>This version of `docker-compose up` uses an override file to modify our services. `docker-compose.dev.yml` we now have an extra PHP container called `esmero-php-debug`. 
 
-   So we have reloaded the containers and now you are ready for Part 2.
+   <br>To stop the containers in the future, run `docker-compose -f docker-compose.yml -f docker-compose.dev.yml down`.
+
+    <br>_(To make these commands easier to remember, consider making bash aliases in your .bashrc file.)_
+    <br>_(If you are running your development on a Linux system, you may need to make a modification to your xdebug configuration file on the esmero-php-dev container. See appendex at the bottom of this page. )
+
+    <br>So we have reloaded the containers and now you are ready for Part 2.
 
 ## Part 2: PHPStorm
 
 1. In PHPStorm, open your `archipelago-deployment` project.
 
 2. Go to `Preferences > Languages & Frameworks > PHP > Debug`. In this window there is an Xdebug section. Use these settings:
```

Comment:
`Preferences > Languages & Frameworks > PHP > Debug` was actually `Settings > PHP > Debug` for me on Ubuntu so maybe a note about possible variations in location depending on OS? Same for 32 below, i.e. `Settings > PHP > Servers`